WcaOnRails/app/webpacker/components/CompetitionsOverview/CompetitionsFilters.js

```diff
@@ -138,15 +138,15 @@ function RegionSelector({ dispatchFilter }) {
       <Dropdown
         search
         selection
-        defaultValue="all"
+        value={region}
```

Comment:
Why are you replacing the `defaultValue` entirely?
Just looking at the code, it feels like we should keep the default value and also set `value={region}` so that it gets overridden if necessary/present.

Reply:
You're not meant to do that; default value is for uncontrolled components, value is for controlled components. Console warning:
> "value" is auto controlled. Specify either defaultValue or value, but not both.

Reply:
Oh, I wasn't aware of that. In this case, nevermind :smile: